gips/0001-gip-process.md

```diff
@@ -31,84 +31,137 @@ The GIP process includes both the stages and acceptance criteria that a proposal
+
+If they prefer, authors may also ask the Editors to assign a number rather than self-assigning.
+
+# After submitting a proposal
+
+Publishing a GIP does not guarantee inclusion in the protocol. Protocol upgrades are implemented via decentralized governance, while the GIP Process is geared towards surfacing and building legitimacy for valuable protocol upgrades that can be considered in that process. Multiple GIPs may be bundled together into a release and be voted on as a single Graph Governance Proposal (GGP).
+
+GGPs may also include protocol upgrades that did not go through the GIP Process, such as bugs that were reported as responsible security disclosures, or enhancements that were too minor to justify a full-fledged GIP.
+
+Once a GIP is published on the GitHub repo and the forum, some time should be left for the community to discuss. Some more complex or controversial proposals may require more time and discussion, while others that are more straightforward may be accepted more quickly, always at the Council's discretion.
+
+After publishing, the authors are encouraged to present the GIPs in one or many of the relevant community calls, for instance, Indexer Office Hours that happen weekly on Discord, or the monthly core developer calls. To request a speaking slot in one of these calls, you can use this form (TODO: add form link) to get in touch with organizers from core dev teams and The Graph Foundation.
+
+# Implementing a proposal
+
+Proposal authors are welcome to begin implementing the changes needed for a proposal at any time before or after submitting the proposal. They do so at their own risk, as there is no guarantee that the community and the Council will move forward with the proposal.
+
+If the author is unable to do the implementation of a GIP, and would like instead for core developers to implement it, this should be expressed in the GIP or the forum discussion, and the author may use this form (TODO: add form link) to request core devs for help implementing a GIP. The author may need to present evidence of community support for the proposal, and core devs' may accept or reject the request at their discretion.
+
+## Proposals requiring off-chain software changes
+
+For proposals requiring changes in the behavior of off-chain components (e.g. Graph Node, Indexer Agent, Indexer Service), changes must be made as Pull Requests to the corresponding repositories (generally under the [graphprotocol org on GitHub](https://github.com/graphprotocol)). Authors are encouraged to collaborate with core developers (and if necessary, the Technical Advisory Board), to ensure the changes meet the quality criteria for each repository. These changes generally do not require explicit Council approval, unless the behavior of the components changes in a way that requires changes to the Feature Matrix or the Arbitration Charter. In particular, Graph Node versions are approved through a GGP.
```

Comment:
we should add links to the Feature Matrix and the Arbitration Charter as this may be the first time authors will first hear about such things. 

https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md

https://github.com/graphprotocol/graph-improvement-proposals/blob/09f464d9797cb8f24f19a9fc8df8b0fb546930c8/gips/0009-arbitration-charter.md#L4